multisigverifier/src/lib.rs

```diff
@@ -47,25 +66,27 @@ pub trait Multisigverifier:
             signature.as_managed_buffer()
         );
 
+        self.is_signature_valid_and_approved(is_bls_valid)
+    }
+
+    fn is_signature_valid_and_approved(&self, is_bls_valid: bool) -> bool {
         let signatures_count = self.signatures().get();
         let bls_pub_keys = self.bls_pub_keys().len() as u32;
+        let minimum_signatures = 2 * bls_pub_keys / 3;
 
-        if is_bls_valid && signatures_count > 2/3 * bls_pub_keys {
+        if is_bls_valid && signatures_count > minimum_signatures {
             return true
         }
 
         false
```

Comment:
Can simply return the condition. i.e.

is_bls_valid && signatures_count > minimum_signatures

Having said that, I wouldn't give a bool arg to this function just to check if it's true, I'd just check it outside the function.

Reply:
done